services/src/server.rs

```diff
@@ -99,7 +102,6 @@ where
                     .handler(http::StatusCode::METHOD_NOT_ALLOWED, render_405),
             )
             .wrap(TracingLogger::<CustomRootSpanBuilder>::new())
-            .wrap(middleware::NormalizePath::trim())
```

Comment:
1. We should be careful removing all normalization, especially since you did not remove that from the pro server.

2. Can't we specify a normalization that makes sense for us?
Moreover, can we put swagger outside of the normalization?

Reply:
1. the normalization regards only the trailing slashes. IMHO it's okay that it works only as we define it in our API, either with, or without a trailing slash, and not the other way around.

2. Not via a middleware, because middleware are application-wide